optaweb-vehicle-routing-docs/src/main/asciidoc/quickstart.adoc

```diff
@@ -0,0 +1,172 @@
+//The application source code depends on Maven and NPM packages that will be downloaded during build.
+//When the application runs it uses third party, public services such as link:https://www.openstreetmap.org/about[OpenStreetMap]
+//to display map tiles and provide search results.
+
+=== Download routing data
+
+The routing engine requires geographical data to calculate the time it takes vehicles to travel between locations.
+You must download and store OSM (OpenStreetMap) data files on the local file system before you run OptaWeb Vehicle Routing.
+
+NOTE: The OSM data files are typically between 100 MB to 1 GB and take time to download so it is a good idea to download the files before building or starting the OptaWeb Vehicle Routing application.
+
+.Procedure
+. Open http://download.geofabrik.de/ in a web browser.
+. Click a region in the *Sub Region* list, for example *Europe*.
+The sub region's page opens.
+. In the *Sub Regions* table, download the OSM file (`.osm.pbf`) for a country, for example Belgium.
+
+=== Create the OptaWeb Vehicle Routing data directory structure
+
+OptaWeb Vehicle Routing reads and writes several types of data on the file system.
+It reads OSM (OpenStreetMap) files from the `openstreetmap` directory, writes a road network graph to the `graphhopper` directory, and persists user data in a directory called `db`.
+Create a new directory dedicated to storing all of these data to make it easier to upgrade to a newer version of OptaWeb Vehicle Routing in the future and continue working with the data you created previously.
+
+.Procedure
+. Create the `openstreetmap` directory in your user account `home` directory, for example:
++
+[source]
+----
+$HOME/.vrp
+└── openstreetmap
+----
+
+. Move all of your downloaded OSM files (files with the extension `.osm.pbf`) to the `openstreetmap` directory.
+
+The rest of the directory structure will be created by the OptaWeb Vehicle Routing application when it runs for the first time.
+After that, your directory structure will look similar to the following example:
+
+// TODO maybe replace this with a screenshot, doesn't look good in PDF.
+[source]
+----
+$HOME/.vrp
+├── db
+│   └── vrp.mv.db
+├── graphhopper
+│   └── belgium-latest
+└── openstreetmap
+    └── belgium-latest.osm.pbf
+----
+
+=== Verify the Java version
+
+// TODO Java 11
+Java SE 8 or higher must be installed on your system before you can use OptaWeb Vehicle Routing.
+It is recommended that you install JDK.
+However, if you have a binary distribution of OptaWeb Vehicle Routing, you only need the Java SE Runtime Environment (JRE).
+But if you want to build OptaWeb Vehicle Routing from source, your system must have the Java SE Development Kit (JDK).
+
+.Procedure:
+. To verify the current Java installation, enter the following command:
++
+[source,shell]
+----
+java -version
+----
+. If necessary, install OpenJDK 8.
+* To install OpenJDK 8 on Fedora, enter the following command:
```

Comment:
In this case I think it's useful to include the command so that readers can skip navigating to another page and looking for it. But the link you shared is useful. I'll add it for non-Fedora users.